app/uk/gov/hmrc/tai/model/tai/JsonExtra.scala

```diff
@@ -16,11 +16,8 @@
 
 package uk.gov.hmrc.tai.model.tai
 
-import org.slf4j.Logger
 import play.api.libs.json._
 
-import scala.util._
-
 object JsonExtra {
```

Comment:
Could move this one into the util package
The 'tai' package seems a bit empty now .. wonder if we could move the TaiRootModel into here .. or just move TaxYear into the root 'model' package and remove this sub package